taxonomies/labels.txt

```diff
@@ -175,6 +175,7 @@ hu:Szabad tartású
 nl:Vrije uitloop
 pt:Criação em liberdade
 ru:свободный выгул, открытый воздух
+label_categories:en: en:Animal welfare
```

Comment:
`en:en:` seems redundant

Reply:
it's the way it is for multilingual taxonomies embedded into other multilingual taxonomies.

Reply:
That's the way it currently works when a property value is an entry in another taxonomy.
e.g. for additives, we have:
additives_classes:en: en:colour

That means that the English value for the additives_classes property is the entry en:colour.

The properties in taxonomy must have a language (and the value can be different based on that language). 

We should do this though:

label_categories:en: en:animal-welfare

Reply:
I'm confused @stephanegigandet in other occurrences I could swear you've asked me to put the real value and not the tag. Should I create a taxonomy of label clusters ?